### PR TITLE
Issue #255 - Create rake task for create indices in ElasticSearch.

### PR DESCRIPTION
**Pull request by nsanta**
Related to issue #255 

- Added rake task for create indices on ElasticSearch for Resource, Network, List and User records. 
- Readme.md Updated.

Also I added the proper namespace to the elastic_search reset tasks.

### CODE REVIEW

File: lib/tasks/elasticsearch.rake [hunk 1]
```diff
@@ -1,46 +1,83 @@
 namespace :elasticsearch do
-  def reset_index(klass)
-    client = Elasticsearch::Client.new(
+  namespace :reset do
+
```
[the remark this answers is not in the document]
Comment (nsanta) [reply lib/tasks/elasticsearch.rake]: Nice watch dog.

File: lib/tasks/elasticsearch.rake [hunk 2]
```diff
@@ -1,46 +1,82 @@
...
+    desc 'Deletes the "list" index and regenerates it with all records currently in List'
+    task list_index: :environment do
+      reset_index(List)
+    end
 
-  desc 'Deletes the "network" index and regenerates it with all records currently in Network'
-  task reset_network_index: :environment do
-    reset_index(Network)
+    desc 'Deletes the "user" index and regenerates it with all records currently in List'
+    task list_index: :environment do
```
Comment (ptrikutam) [inline lib/tasks/elasticsearch.rake]: Looks like a typo here, should this be `user_index`?

Comment (nsanta) [reply lib/tasks/elasticsearch.rake]: Funny. The typo exists before this PR 😆

**Comment by nsanta** [comment]
I can't add the rake task to the setup script because the server needs to be started with foreman before run the task.

File: lib/tasks/elasticsearch.rake (hunk 2 above)
Comment (nsanta) [reply lib/tasks/elasticsearch.rake]: Fixed.

File: lib/tasks/elasticsearch.rake [hunk 3]
```diff
@@ -1,46 +1,82 @@
...
+        klass.import
+      rescue Faraday::ConnectionFailed => e
+        ap e
+        ap 'Indexing records one at a time...'
+        klass.all.each { |r| r.__elasticsearch__.index_document }
+      end
     end
 
-    klass.__elasticsearch__.create_index! force: true
+    desc 'Deletes the "resource", "network", "user" and "list" indices and regenerates it with all records'
```
[the remark this answers is not in the document]
Comment (nsanta) [reply lib/tasks/elasticsearch.rake]: @ptrikutam How I can skip this validation on hound?

**Comment by ptrikutam** [comment]
@nsanta thanks for fixing the typo.

> @ptrikutam I can't add the rake task to the setup script because the server needs to be started with foreman before run the task.

Gotcha. OK, no worries. Can you update this wording:

> If you get errors from ElasticSearch on startup, try running the following command in the Console:

to 

> You will need to run the following while you have foreman running in another terminal window, otherwise you will get ElasticSearch errors:

Or something to that effect.

**Pull request merged**